tests/test_organization.py

```diff
@@ -483,15 +500,15 @@ def test_organization_identifiers(self):
 
     def test_organization_identifiers_update(self):
         with open(os.path.join(
-            os.path.dirname(__file__),
-            'organization-example-gastro.json'), 'r') as fhir_data:
+                os.path.dirname(__file__),
+                'organization-example-gastro.json'), 'r') as fhir_data:
```

Comment:
Can you de-indent these and move the `) as fhir_data:` to a new line, aligned with `with`?